### DISCUSSION TITLE
2021-06-30 Weekly Meeting 10 Agenda / Discussion

### DISCUSSION THREAD

**Discussion by robertlong** [omigroup/omigroup#52] [Weekly Meetings]
Weekly Meeting 10 is happening on Wednesday 2021-06-30 from 9:00 AM PDT - 10:00 AM PDT

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).

We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR Discord](https://discord.gg/NJtT9grz5E).

## Agenda
- [OMI Open Collective Fundraising](https://opencollective.com/omigroup)
- [OMI Website](https://omigroup.org)

**Comment by mrmetaverse** [comment]
Meeting notes: https://hackmd.io/oJ1dkVkwRAaVmEWbU-WYtQ
